library/common-library/DEVELOPERS.md

```diff
@@ -0,0 +1,615 @@
+# Functions/templates documented for chart writers
+Here is some rough documentation separated by the file that contains the function, the function
+name and how to use it. We are not covering functions that start with `_` (e.g.
+`newrelic.common.license._licenseKey`) because they are used internally by this library for
+other helpers. Helm does not have the concept of "public" or "private" functions/templates so
+this is a convention of ours.
+
+## _naming.tpl
+These functions are used to name objects.
+
+### `newrelic.common.naming.name`
+This is the same as the idiomatic `CHART-NAME.name` that is created when you use `helm create`.
+
+It honors `.Values.nameOverride`.
+
+Usage:
+```mustache
+{{ include "newrelic.common.naming.name" . }}
+```
+
+### `newrelic.common.naming.fullname`
+This is the same as the idiomatic `CHART-NAME.fullname` that is created when you use `helm create`
+
+It honors `.Values.fullnameOverride`.
+
+Usage:
+```mustache
+{{ include "newrelic.common.naming.fullname" . }}
+```
+
+### `newrelic.common.naming.chart`
+This is the same as the idiomatic `CHART-NAME.chart` that is created when you use `helm create`.
+
+It is mostly useless for chart writers. It is used internally for templating the labels but there
+is no reason to keep them "private".
```

Comment:
them?

Reply:
Fixed in [#6216d9f](https://github.com/newrelic/helm-charts/pull/797/commits/6216d9f3f09fedf3717c1b7a457a887fd8230371)